data/types/pumpstatus/battery.go

```diff
@@ -0,0 +1,40 @@
+package pumpstatus
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	MinBatteryPercentage = 0.0
+	MaxBatteryPercentage = 1.0
+)
+
+type Battery struct {
+	Unit  *string  `json:"unit,omitempty" bson:"unit,omitempty"`
```

Comment:
Should have enums for units. For example, `percent` and `voltage`. If we're only considering `percent`, then we can get rid of `Unit`, and `Value` wouldn't be a float (I don't think we'd be measuring parts of a percent)

Reply:
I believe that percentage is the only item we will need - so deleting the unit.  Then - the question becomes - should be just have a batteryPercentage on pumpStatus and eliminate the object?  For now - I am leaving it as an object in case we want to add more elements (such as model, type ...)